setup.cfg

```diff
@@ -8,6 +8,10 @@ BLANK_LINE_BEFORE_NESTED_CLASS_OR_DEF = False
 max-line-length = 100
 ignore = E301
 
+[flake8]
+max-line-length = 100
```

Comment:
how about 120?

Reply:
I prefer 120 but I'm not sure if this is too far from what people are used to (80)

Reply:
let's do 120. We are the 👑